app/assets/javascripts/bootstrap.js.coffee

```diff
@@ -1,11 +1,3 @@
-#
-#= require twitter/bootstrap
-#= require datetimepicker/bootstrap-datetimepicker
-#= require bootstrap-fileupload
-#= require bootstrap-multiselect
```

Comment:
What's actually the reason for removing those? You didn't mention that in the commit message